src/mongo/db/exec/multi_plan.h

```diff
@@ -198,10 +203,14 @@ class MultiPlanStage final : public PlanStage {
     // uses -1 / kNoSuchPlan when best plan is not (yet) known
     int _bestPlanIdx;
 
-    // index into _candidates, of the backup plan for sort
+    // index into _candidates, of the backup of the plan competition
```

Comment:
This still doesn't read particularly well. How about re-phrasing this to
` The index within '_candidates' of the non-blocking backup plan which can be used if a blocking plan fails. This is set to 'kNoSuchPlan' if there is no backup plan, or when it is not yet known.`? I'd recommend a similar re-wording for the comment on '_originalWinningPlanIndex' below, with a mention that this is used to remember which plan one in the case that a backup plan took over.

Reply:
It looks like this still hasn't been addressed?